src/components/Communicator/CommunicatorDialog/CommunicatorDialogBoardItem.component.js

```diff
@@ -387,6 +389,68 @@ class CommunicatorDialogBoardItem extends React.Component {
       </IconButton>
     );
 
+    const PublicBoardInfo = () => {
+      return (
+        <Dialog
+          onClose={this.handleDialogClose.bind(this)}
+          aria-labelledby="board-info-title"
+          open={this.state.openBoardInfo}
+          className="CommunicatorDialog__boardInfoDialog"
+        >
+          <DialogTitle
+            id="board-info-title"
+            onClose={this.handleDialogClose.bind(this)}
+          >
+            {board.name}
+          </DialogTitle>
+          <DialogContent>
+            <DialogContentText>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardInfoName)}:</b>{' '}
+                {board.name}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardInfoAuthor)}:</b>{' '}
+                {board.author}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardInfoLocale)}:</b>{' '}
+                {board.locale}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardDescription)}:</b>{' '}
+                {board.description}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardInfoDate)}:</b>{' '}
+                {moment(board.lastEdited).format('DD/MM/YYYY')}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardInfoTiles)}:</b>{' '}
+                {board.tiles.length}
+              </Typography>
+              <Typography variant="body1" gutterBottom>
+                <b>{intl.formatMessage(messages.boardInfoId)}:</b> {board.id}
+              </Typography>
+              {!isCordova() && (
+                <Typography variant="body1" gutterBottom>
+                  <b>{intl.formatMessage(messages.boardInfoUrl)}:</b>{' '}
+                  <a href={boardUrl} target="_blank" rel="noopener noreferrer">
+                    {boardUrl}
+                  </a>
+                </Typography>
+              )}
+            </DialogContentText>
+          </DialogContent>
+          <DialogActions>
+            <Button onClick={this.handleDialogClose.bind(this)} color="primary">
+              {intl.formatMessage(messages.close)}
+            </Button>
+          </DialogActions>
+        </Dialog>
+      );
+    };
```

Comment:
Please in the next PRs avoid make changes that are strict of the PR. For now is okay because is small and is your first PR and this changes are correct.